docs/Bootloaders.md

```diff
@@ -382,13 +382,14 @@ make flash FLASH_DEVICE=/dev/ttyACM0
 It may be necessary to manually enter the bootloader, this can be done by
 setting "boot 0" low and "boot 1" high.  On the SKR Mini E3 "Boot 1" is
 not available, so it may be done by setting pin PA2 low if you flashed
-"hid_btt_skr_mini_e3.bin".  This pin is labeld "TX0" on the TFT header in
+"hid_btt_skr_mini_e3.bin".  This pin is labeled "TX0" on the TFT header in
 the SKR Mini E3's "PIN" document. There is a ground pin next to PA2
 which you can use to pull PA2 low.
 
 ### STM32F103/STM32F072 with MSC bootloader
 
-The [MSC bootloader](https://github.com/Telekatz/MSC-stm32f103-bootloader) is a driverless bootloader capable of flashing over USB.
+The [MSC bootloader](https://github.com/Telekatz/MSC-stm32f103-bootloader) is a
```

Comment:
This was fine with 75 line.

Reply:
@qwewer0 thanks for review, restored the line.